src/NzbDrone.Common/Http/HttpRequestBuilder.cs

```diff
@@ -232,6 +234,13 @@ public virtual HttpRequestBuilder Resource(string resourceUrl)
             return this;
         }
 
+        public virtual HttpRequestBuilder KeepAlive()
```

Comment:
this feels a bit strange, I think `KeepAlive(true)` `KeepAlive(false)` would be nicer.